contributions/catalog/webapps/templates/contribute/contribute.html

```diff
@@ -1668,8 +1668,33 @@ <h3 id="contact">Contact Details</h3>
                     <br>
                 </div>
             </div>
+
+            <!-- Cancel Confirmation modal -->
+            <div class="modal" id="CancelModal" role="dialog">
```

Comment:
I think it would be better to use hyphenated words for IDs and CSS class names. Could you please use cancel-modal or something similar? 

I see that there is also a "myModal" from the existing code. I would suggest changing its name too to something more meaningful. Thanks.

Reply:
updated

Reply:
Thanks.